resources/assets/uad_lists.json

```
@@ -17630,6 +17630,15 @@
     "labels": [],
     "removal": "Recommended"
   },
+  {
+    "id": "com.google.android.safetycenter.resources",
+    "list": "Google",
+    "description": "Google safety center anti-virus spyware\n(https://safety.google). Disables the 'Please enable google play\nprotect' prompt when installing from F-Droid after disabling it in the\nPlay Store. Probably also disables gmail and browser malware notifications.\n\nUse a libre spam-blocking and DNS-blocking solution instead.",
```

```suggestion
    "description": "Google Safety Center (https://safety.google). Disables the 'Please enable Google Play/Protect' prompt when installing from F-Droid after disabling it in the Play Store. Probably also disables Gmail and browser malware notifications.\nUse a libre spam-blocking and DNS-blocking solution instead.",
```
I think you're using `\n` wrong. It's not a slash, but a new line, see https://www.w3schools.com/c/c_newline.php.

---

Looks good to me.

---

Should I use \n after a certain number of characters, or just let UAD decide how to wrap my text?

---

You'll have to decide that yourself. I usually insert a line break before a URL or after ~2 sentences.